cmd/dep/gom_importer_test.go

```diff
@@ -0,0 +1,219 @@
+	want := h.GetTestFileString(goldenFile)
+	if want != got {
+		if *test.UpdateGolden {
+			if err := h.WriteTestFile(goldenFile, got); err != nil {
+				t.Fatalf("%+v", errors.Wrapf(err, "Unable to write updated golden file %s", goldenFile))
+			}
+		} else {
+			t.Fatalf("expected %s, got %s", want, got)
+		}
+	}
+}
+
+func TestGomConfig_ConvertProject(t *testing.T) {
+	h := test.NewHelper(t)
+	defer h.Cleanup()
+
+	ctx := newTestContext(h)
+	sm, err := ctx.SourceManager()
+	h.Must(err)
+	defer sm.Release()
+
+	g := newGomImporter(discardLogger, true, sm)
+	g.goms = []gomPackage{
+		{
+			name: "github.com/sdboyer/deptest",
+			options: map[string]interface{}{
+				"commit": "ff2948a2ac8f538c4ecd55962e919d1e13e74baf",
+			},
+		},
+	}
+
+	manifest, lock, err := g.convert("")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	d, ok := manifest.Constraints["github.com/sdboyer/deptest"]
+	if !ok {
+		t.Fatal("Expected the manifest to have a dependency for 'github.com/sdboyer/deptest' but got none")
+	}
+
+	v := d.Constraint.String()
+	if v != "ff2948a2ac8f538c4ecd55962e919d1e13e74baf" {
+		t.Fatalf("Expected manifest constraint to be %q got %q", "ff2948a2ac8f538c4ecd55962e919d1e13e74baf", v)
+	}
+
+	p := lock.P[0]
+	if p.Ident().ProjectRoot != "github.com/sdboyer/deptest" {
+		t.Fatalf("Expected the lock to have a project for 'github.com/sdboyer/deptest' but got '%s'", p.Ident().ProjectRoot)
+	}
+
+	lv := p.Version()
+	lpv, ok := lv.(gps.PairedVersion)
+	if !ok {
+		t.Fatalf("Expected locked version to be PairedVersion but got %T", lv)
+	}
+
+	rev := lpv.Underlying()
+	if rev != "ff2948a2ac8f538c4ecd55962e919d1e13e74baf" {
+		t.Fatalf("Expected locked revision to be 'ff2948a2ac8f538c4ecd55962e919d1e13e74baf', got %s", rev)
+	}
+}
+
+func TestGomConfig_ConvertProject_EmptyComment(t *testing.T) {
+	h := test.NewHelper(t)
+	defer h.Cleanup()
+	h.TempDir("src")
+
+	ctx := newTestContext(h)
+	sm, err := ctx.SourceManager()
+	h.Must(err)
+	defer sm.Release()
+
+	g := newGomImporter(discardLogger, true, sm)
+	g.goms = []gomPackage{
+		{
+			name: "github.com/sdboyer/deptest",
+			options: map[string]interface{}{
+				"branch": "0.8.0",
+			},
+		},
+	}
+
+	manifest, lock, err := g.convert("")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	d, ok := manifest.Constraints["github.com/sdboyer/deptest"]
+	if !ok {
+		t.Fatal("Expected the manifest to have a dependency for 'github.com/sdboyer/deptest' but got none")
+	}
+
+	v := d.Constraint.String()
+	if v != "^0.8.0" {
+		t.Fatalf("Expected manifest constraint to be %q got %q", "^0.8.0", v)
+	}
+
+	p := lock.P[0]
+	if p.Ident().ProjectRoot != "github.com/sdboyer/deptest" {
+		t.Fatalf("Expected the lock to have a project for 'github.com/sdboyer/deptest' but got '%s'", p.Ident().ProjectRoot)
+	}
+
+	rev := p.Version().String()
+	if rev != "0.8.0" {
+		t.Fatalf("Expected locked revision to be 'ff2948a2ac8f538c4ecd55962e919d1e13e74baf', got %s", rev)
+	}
+}
+
+func TestGomConfig_ProjectExistsInLock(t *testing.T) {
```

Comment:
This is a duplicate test of https://github.com/golang/dep/blob/cc112746e402919651ea7fbfca29fb031ba98e37/cmd/dep/godep_importer_test.go#L305 and can be removed.